components/AdminScoringForm.tsx

```diff
@@ -0,0 +1,231 @@
+            </label>
+            <TextField.Root
+              id="age16Score"
+              name="age16Score"
+              type="number"
+              min={0.0}
+              max={10.0}
+              step={0.1}
+              className="flex-grow"
+              disabled={!adminScoringData?.age16ExamType}
+              required={!!adminScoringData?.age16ExamType}
+              defaultValue={adminScoringData?.age16Score}
+            />
+          </Flex>
+
+          <Heading as="h3" size="3">
+            Age 18 exam
+          </Heading>
+          <Flex gap="1" align="center">
+            <label className="w-1/5" htmlFor="age18ExamType">
+              Type:
+            </label>
+            <input name="age18ExamType" type="hidden" value={adminScoringData?.age18ExamType} />
+            <Dropdown
+              values={Object.keys(QualificationType18)}
+              currentValue={adminScoringData?.age18ExamType}
+              onValueChange={(value) =>
+                setAdminScoringData(
+                  adminScoringData && {
+                    ...adminScoringData,
+                    age18ExamType: value as QualificationType18
+                  }
+                )
+              }
+              className="flex-grow"
+            />
+          </Flex>
+          <Flex gap="1" align="center">
+            <label className="w-1/5" htmlFor="age18Score">
+              Score:
+            </label>
+            <TextField.Root
+              id="age18Score"
+              name="age18Score"
+              type="number"
+              min={0.0}
+              max={10.0}
+              step={0.1}
+              className="flex-grow"
+              disabled={!adminScoringData?.age18ExamType}
+              required={!!adminScoringData?.age18ExamType}
+              defaultValue={adminScoringData?.age18Score}
+            />
+          </Flex>
+          <Flex gap="2" direction="column">
+            <Heading asChild size="3">
+              <label htmlFor="motivationAssessments">Motivational Assessments</label>
+            </Heading>
+            <TextField.Root
+              id="motivationAssessments"
+              name="motivationAssessments"
+              type="number"
+              min={0.0}
+              max={10.0}
+              step={0.1}
+              defaultValue={adminScoringData?.imperialReview?.motivationAssessments}
+            />
+          </Flex>
+
+          <Flex gap="2" direction="column">
+            <Heading asChild size="3">
+              <label htmlFor="extracurricularAssessments">Extracurricular Assessments</label>
+            </Heading>
+            <TextField.Root
+              id="extracurricularAssessments"
+              name="extracurricularAssessments"
+              type="number"
+              min={0.0}
+              max={10.0}
+              step={0.1}
+              defaultValue={adminScoringData?.imperialReview?.extracurricularAssessments}
+            />
+          </Flex>
```

Comment:
May I suggest a standardised `LabelledFormField`? You can pass the actual input as `children`.

```suggestion
          <Flex gap="2" direction="column">
             <label>
                <Text as="div" size="2" weight="bold">Extracurricular Assessments</Text>
                <TextField.Root
                  id="extracurricularAssessments"
                  name="extracurricularAssessments"
                  type="number"
                  min={0.0}
                  max={10.0}
                  step={0.1}
                  defaultValue={adminScoringData?.imperialReview?.extracurricularAssessments}
                />
            </label>
          </Flex>
```

Reply:
Resolved in [66d5a40](https://github.com/imperial/ug-admissions/pull/6/commits/66d5a4091670cda4acea0daf16b68da64da1399a)

![image](https://github.com/user-attachments/assets/d22fb244-f88b-402e-87ae-0ee36dcd9263)
*Admin scoring form*